results/google__text-embedding-005/external/AmazonCounterfactualClassification.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dataset_revision": "e8379541af4e31359cca9fbcf4b00f2671dba205",
+  "mteb_dataset_name": "AmazonCounterfactualClassification",
+  "mteb_version": "1.0.3.dev0",
```

Comment:
This is quite an old version of MTEB. In the future, we might outdate results from before v1.11.0 (due to file format and a few notable bug fixes).

I would recommend rerunning this using the latest version.

Reply:
I will leave this as a TODO in a future eval since it require some effort to run the latest version with our infrastructure properly.